xbmc/addons/AddonInstaller.cpp

```diff
@@ -818,10 +818,11 @@ bool CAddonInstallJob::Install(const std::string &installFrom, const RepositoryP
           AddonPtr dependencyToInstall;
 
           // origin of m_addon is empty at least if an addon is installed for the first time
-          // we need to override "parentRepoId" in this case
+          // we need to override "parentRepoId" in this case when the addon is installed
+          // from a repo
```

Comment:
maybe `we need to override "parentRepoId" if the passed in repo is valid`. this error didn't happen when installed from a repo i think

Reply:
Done